src/main/java/seedu/address/model/profile/Descriptor.java

```diff
@@ -9,8 +9,9 @@
   * Guarantees: immutable; non-empty; name is valid as declared in {@link #isValidDescriptor(String)}
   */
 public class Descriptor {
-    public static final String MESSAGE_CONSTRAINTS = "descriptors names should be alphanumeric and non-empty";
-    public static final String VALIDATION_REGEX = "^[A-Za-z0-9]+$";
+    public static final String MESSAGE_CONSTRAINTS = "descriptors should be alphanumeric,"
+        + "punctuations limited to .,?!:; and non-empty";
+    public static final String VALIDATION_REGEX = "^[A-Za-z0-9 .,!?:;]*+$";
 
```

Comment:
not totally sure how regex works but from the description are you rejecting # and +?

Reply:
I was thinking for descriptors - equivalent of the skills in internship should just be a non-empty string! Because I don't think there are any wrong formats that we should disallow them from inputting?

Reply:
@seanjyjy yeah I was thinking you could always type c sharp or cpp  for those cases .
@keanecjy Good point, I am ambivalent about this. But could the same be said for Names as well, maybe we can consider a char limit for that?

Reply:
oh, but I think keane/shawn side is allowing the use of those though! it might be confusing for the user ?

Reply:
I think it would be better to put in C# instead of C sharp? Since its easier to put in. Not sure about the char limit though, since some companies may have very long names?